### ISSUE TITLE
Magento 2 Upgrade from 2.4.3-p2 to 2.4.4

### ISSUE THREAD

**Issue by tim-breitenstein-it** [wolfsellers/module-enabledisabletfa#3]
```
$ php bin/magento setup:di:compile
Compilation was started.
Repositories code generation... 1/9 [===>------------------------]  11% < 1 sec 127.0 MiBClass 'Magento\TwoFactorAuth\TestFramework\ControllerActionPredispatch' not found#0 /var/www/magento-244/vendor/composer/ClassLoader.php(571): include()
#1 /var/www/magento-244/vendor/composer/ClassLoader.php(428): Composer\Autoload\includeFile()
#2 [internal function]: Composer\Autoload\ClassLoader->loadClass()
#3 [internal function]: spl_autoload_call()
#4 /var/www/magento-244/setup/src/Magento/Setup/Module/Di/Code/Reader/ClassesScanner.php(134): class_exists()
#5 /var/www/magento-244/setup/src/Magento/Setup/Module/Di/Code/Reader/ClassesScanner.php(117): Magento\Setup\Module\Di\Code\Reader\ClassesScanner->includeClass()
#6 /var/www/magento-244/setup/src/Magento/Setup/Module/Di/Code/Reader/ClassesScanner.php(87): Magento\Setup\Module\Di\Code\Reader\ClassesScanner->extract()
#7 /var/www/magento-244/setup/src/Magento/Setup/Module/Di/App/Task/Operation/RepositoryGenerator.php(61): Magento\Setup\Module\Di\Code\Reader\ClassesScanner->getList()
#8 /var/www/magento-244/setup/src/Magento/Setup/Module/Di/App/Task/Manager.php(56): Magento\Setup\Module\Di\App\Task\Operation\RepositoryGenerator->doOperation()
#9 /var/www/magento-244/setup/src/Magento/Setup/Console/Command/DiCompileCommand.php(216): Magento\Setup\Module\Di\App\Task\Manager->process()
#10 /var/www/magento-244/vendor/symfony/console/Command/Command.php(255): Magento\Setup\Console\Command\DiCompileCommand->execute()
#11 /var/www/magento-244/vendor/symfony/console/Application.php(1009): Symfony\Component\Console\Command\Command->run()
#12 /var/www/magento-244/vendor/symfony/console/Application.php(273): Symfony\Component\Console\Application->doRunCommand()
#13 /var/www/magento-244/vendor/magento/framework/Console/Cli.php(115): Symfony\Component\Console\Application->doRun()
#14 /var/www/magento-244/vendor/symfony/console/Application.php(149): Magento\Framework\Console\Cli->doRun()
#15 /var/www/magento-244/bin/magento(23): Symfony\Component\Console\Application->run()
#16 {main}
```

**Comment by mageprashant** [comment]
@jtomaszewski Same issue.

**Comment by Green2Matter** [comment]
Exactly the same in my test instance. Is extension abandoned?

**Comment by gdvisser** [comment]
This module is a good alternative: https://github.com/markshust/magento2-module-disabletwofactorauth

**Comment by Rus0** [comment]
It was resolved with the PR #4
